### PR TITLE
style: External portals within search results

### PR DESCRIPTION
**Pull request by ianjon3s**
## What does this PR do?

- Reduces visual weight of external portals within search results
- Uses a `<a>` tag for the links, allowing for opening in new tabs (right/ctrl click as native behaviour)

**Existing (left) vs updated (right):**
image

### CODE REVIEW

File: editor.planx.uk/src/pages/FlowEditor/components/Sidebar/Search.tsx
Comment (ianjon3s) [inline editor.planx.uk/src/pages/FlowEditor/components/Sidebar/Search.tsx:207]: @DafyddLlyr there also appears to have been some auto-linting in play here

**Review by DafyddLlyr — APPROVED** [review]
Nice! If it's ok with you we'll hold off merging this to `dp/fuse-js` just now until that PR has been approved and merged, then we can repoint this to `main` in a couple of days 👍

**Pull request merged**
